DOCUMENTATION.md

```diff
@@ -401,19 +401,23 @@ File.open("path/to/pdf_file", "rb") do |pdf_file|
 end
 ```
 
+#### postage (optional)
+
+You can select first or second class postage for your precompiled letter. Set the value to `first` for first class, or `second` for second class. If you do not pass in this argument, the postage will default to second class.
+
+
 ### Response
 
-If the request to the client is successful, the client returns a `Notifications::Client:ResponseNotification` object. In the example shown in the [Method section](/ruby.html#send-a-pre-compiled-letter-method), the object is named `precompiled_letter`.
+If the request to the client is successful, the client returns a `Notifications::Client:ResponsePrecompiledLetter` object. In the example shown in the [Method section](/ruby.html#send-a-pre-compiled-letter-method), the object is named `precompiled_letter`.
 
 You can then call different methods on this object to return the requested information.
 
 |Method|Information|Type|
 |:---|:---|:---|
 |`precompiled_letter.id`|Notification UUID|String|
 |`precompiled_letter.reference`|`reference` argument|String|
-|`precompiled_letter.content`|Always `nil`|nil|
-|`precompiled_letter.template`|Always `nil`|nil|
-|`precompiled_letter.uri`|Always `nil`|nil|
+|`precompiled_letter.postage`|`postage` argument|String or nil|
```

Comment:
We could just say 'String' here in the type, since it will never be nil for precompiled letters

Reply:
ahh, that's true!